import-export-cli/integration/throttlePolicy_test.go

```diff
@@ -0,0 +1,454 @@
+			throttlePolicy, _ := testutils.PolicyStructToMap(newPolicy)
+			args := &testutils.PolicyImportExportTestArgs{
+				CtlUser:  testutils.Credentials{Username: user.CtlUser.Username, Password: user.CtlUser.Password},
+				Policy:   throttlePolicy,
+				Type:     customThrottlePolicyFlag,
+				DestAPIM: prod,
+				Update:   true,
+			}
+			adminUsername := adminservices.AdminUsername
+			adminPassword := adminservices.AdminPassword
+			testutils.ValidateThrottlePolicyImportUpdate(t, args, adminUsername, adminPassword, apim.CustomThrottlePolicyType)
+		})
+	}
+}
+
+// Get  Throttle Policy List APICTL output in JsonPretty format and check whether all policies are included
+func TestGetThrottlePoliciesListWithJsonPrettyFormat(t *testing.T) {
+	//devops users don't have access to view throttling policies
+	for _, user := range testCaseUsers[:2] {
+		t.Run(user.Description, func(t *testing.T) {
+
+			dev := GetDevClient()
+
+			args := &testutils.PolicyImportExportTestArgs{
+				CtlUser: testutils.Credentials{Username: user.CtlUser.Username, Password: user.CtlUser.Password},
+				SrcAPIM: dev,
+			}
+			testutils.ValidateThrottlePoliciesList(t, true, args)
+		})
+	}
+}
+
+// Import failure with a corrupted file
+func TestImportInvalidThrottlePolicyFile(t *testing.T) {
+
+	for _, user := range testCaseUsers {
+		t.Run(user.Description, func(t *testing.T) {
+			prod := GetProdClient()
+
+			args := &testutils.PolicyImportExportTestArgs{
+				CtlUser:  testutils.Credentials{Username: user.CtlUser.Username, Password: user.CtlUser.Password},
+				DestAPIM: prod,
+				Update:   true,
+			}
+			testutils.ValidateThrottlePolicyImportFailureWithCorruptedFile(t, args)
+		})
+	}
+}
+
+// Export an invalid throttling policy
+func TestExportInvalidThrottlePolicy(t *testing.T) {
+
+	for _, user := range testCaseUsers {
+		t.Run(user.Description, func(t *testing.T) {
+			dev := GetDevClient()
+
+			args := &testutils.PolicyImportExportTestArgs{
+				CtlUser: testutils.Credentials{Username: user.CtlUser.Username, Password: user.CtlUser.Password},
+				SrcAPIM: dev,
+			}
+			testutils.ValidateThrottlePolicyExportFailure(t, args)
+		})
+	}
+}
+
+// Export a Throttling Policy from one environment and import to another environment without type flag
+func TestExportImportThrottlePolicyWithoutTypeFlag(t *testing.T) {
+
+	for _, user := range testCaseUsers {
+		t.Run(user.Description, func(t *testing.T) {
+			dev := GetDevClient()
+			prod := GetProdClient()
+
+			newPolicy := testutils.AddNewThrottlePolicy(t, dev, user.Admin.Username, user.Admin.Password, apim.AdvancedThrottlePolicyType)
+			throttlePolicy, _ := testutils.PolicyStructToMap(newPolicy)
+			args := &testutils.PolicyImportExportTestArgs{
+				CtlUser:  testutils.Credentials{Username: user.CtlUser.Username, Password: user.CtlUser.Password},
+				Policy:   throttlePolicy,
+				SrcAPIM:  dev,
+				DestAPIM: prod,
+				Update:   false,
+			}
+			adminUsername := adminservices.AdminUsername
```

Comment:
shall we use superAdminUser and superAdminPassword variables directly without using adminservices.AdminUsername and adminservices.AdminPassword.

Refer [1].

[1] https://github.com/wso2/product-apim-tooling/blob/master/import-export-cli/integration/api_test.go#L84

Reply:
Fixed with 479b4168267adb70bde906548a563dade6a07df8